### PR TITLE
Don’t show Profil and My Activities for simple user

### PR DESCRIPTION
**Pull request by blazarecki**

### CODE REVIEW

File: src/DataFixtures/ORM/LoadUserData.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            'password' => 'enmarche',
+            'email' => 'simple-user@example.ch',
+            'gender' => 'female',
+            'first_name' => 'Simple',
+            'last_name' => 'User',
+            'address' => PostAddress::createForeignAddress('CH', '8057', 'Zürich', '32 Zeppelinstrasse', 47.3950786, 8.5361402),
+            'birthdate' => '1972-11-24',
+        ]);
+
+        PropertyReplacer::replaceProperty($user1, 'adherent', false);
```
Comment (B-Galati) [inline src/DataFixtures/ORM/LoadUserData.php]: Overkill, just add a parameters to the factory to disable the `join()` call in `createFromArray` method.

Comment (HeahDude) [reply src/DataFixtures/ORM/LoadUserData.php]: Why? we should not break the encapsulation of the entity because of this line in fixtures loading.

Comment (B-Galati) [reply src/DataFixtures/ORM/LoadUserData.php]: `\AppBundle\Membership\AdherentFactory::createFromArray` is only used for tests (which questionable too). So we can just add a new parameters to this method to conditionally called `$adherent->join();`. It's not breaking encapsulation.

Comment (HeahDude) [reply src/DataFixtures/ORM/LoadUserData.php]: >is only used for tests (which is questionable)
Yup, like many factories, this is in my list of improvements ;).

Comment (blazarecki) [reply src/DataFixtures/ORM/LoadUserData.php]: I fix it, but for me we can break the encapsulation for testing purpose.

Comment (B-Galati) [reply src/DataFixtures/ORM/LoadUserData.php]: Sure if you have a very good reason 👍

**Pull request merged**